File: fe2-android/app/src/main/java/com/github/dedis/popstellar/model/event/EventState.java
package com.github.dedis.popstellar.model.event;

/**
 * Indicates in which state the event is This is useful for example for displaying whether an event
 * is currently taking place or not Other states could be added if needed
 */
public enum EventState {
  CREATED,
  OPENED,
  CLOSED,
  RESULTS_READY
}

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/event/EventState.kt
package com.github.dedis.popstellar.model.objects.event

/**
 * Indicates in which state the event is This is useful for example for displaying whether an event
 * is currently taking place or not Other states could be added if needed
 */
enum class EventState {
  CREATED,
  OPENED,
  CLOSED,
  RESULTS_READY
}
